keytree/tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import xml.etree.ElementTree
+import lxml.etree
+
+import pytest
+
+
+@pytest.fixture(params=[lxml.etree, xml.etree.ElementTree])
+def etree(request):
+    return request.param
```

Comment:
Tests that are given this fixture are run twice, once using lxml and once using the stdlib?

Reply:
Can you make lxml an optional testing dependency? As far as API development goes, we don't really need to test that this package works with lxml... it's lxml's job to implement the element tree API. But yes, for CI, we should check with lxml.

Reply:
With the latest commits, each test will be run twice if `lxml` is available. If it isn't, `pytest` will report the `lxml` half of the tests as skipped but not as failed.